test/lib/warn-upload-overwrite.spec.js

```diff
@@ -157,6 +160,137 @@ describe('warn-upload-overwrite', () => {
         assert.equal(0, readline.keyInSelect.callCount);
       });
     });
+
+    it('prompts the user if a compressible doc type has changes', () => {
+      sinon.stub(readline, 'keyInSelect').returns(-1);
+      sinon.stub(readline, 'keyInYN').returns(true);
+      sinon.stub(environment, 'apiUrl').get(() => 'http://admin:pass@localhost:35423/medic');
+      sinon.stub(request, 'get').returns({'compressible_types':'text/*, application/*','compression_level':'8'});
```

Comment:
Change this to `sinon.stub(request, 'get').resolves(...)`. This request is async.